src/RequestBuilder/ItemPropertiesSetupRequestBuilder.php

```diff
@@ -7,10 +7,8 @@
 use Lmc\Matej\Model\Command\ItemPropertySetup;
 use Lmc\Matej\Model\Request;
 
-class ItemPropertiesSetupRequestBuilder extends AbstractRequestBuilder
+class ItemPropertiesSetupRequestBuilder extends ItemPropertiesGetRequestBuilder
```

Comment:
What do you think about this? (The [comment](https://github.com/lmc-eu/matej-client-php/pull/53#discussion_r154908592) got "outdated" by guthub after the rebase...)

> If the inheritance is there just because of the endpoint path, I'd not introduce it at all and rather have the endpoint path copied on two places. "Setup request builder" is from my POV not specific version of "Get request builder". It would only make sense IMHO to have `AbstractItemPropertiesRequestBuilder` with two childs, but would be a bit overengineered :).

Reply:
This is a very good point; I was thinking from the perspective of REST endpoints - and not from the perspective of RequestBuilders.

Will update code in a sec.